PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -313,12 +313,6 @@ def nanoAOD_customizeCommon(process):
     _unmodifiedPatTauMVAIDsSeq = process.patTauMVAIDsSeq.copy()
     _unmodifiedTauIDSources = process.slimmedTausUpdated.tauIDSources.clone()
     process = nanoAOD_addTauIds(process)
-    for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94X2016, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_102Xv1:
```

Comment:
Hello,
I am not really sure if it makes sense to enable DeepTauID for `run2_miniAOD_80XLegacy` era. It is as for it the content of MiniAOD is quite different than of 94XMiniAODv2 (2017) used to train DeepTauID (primary min track pT used to store detailed track info).
=> My proposal is to keep DeepTauID disabled for `run2_miniAOD_80XLegacy`.

Reply:
ok indeed it was crashing at the moment... I disable it

Reply:
OK, thanks. 
BTW,  DeepTauID has never been tested with 94XMiniAODv1 samples, i.e. with `run2_nanoAOD_94XMiniAODv1` era modifier. I suppose that it will work fine, but if there are any issues I propose to disable DeepTauID also for this era. Anyway, I do not expect anyone interested with running it with those quite old samples already superseded by 94XMiniAODv2 ones.